ui/e2e/services/list.spec.ts

```diff
@@ -6,13 +7,41 @@ import {
 import { createNamespace, deleteNamespace } from "../utils/namespace";
 import { expect, test } from "@playwright/test";
 
-import { createFile } from "e2e/utils/files";
 import { encode } from "js-base64";
 import { headers } from "e2e/utils/testutils";
 import { patchFile } from "~/api/files/mutate/patchFile";
 
 let namespace = "";
 
+const systemNamespace = "system";
+const systemServiceName = "http-service.yaml";
+let systemNamespaceAlreadyExists = false;
+
+test.beforeAll(async () => {
+  try {
+    await createNamespace(systemNamespace);
+  } catch (e) {
+    systemNamespaceAlreadyExists = true;
+  }
+
+  await createFile({
+    name: systemServiceName,
+    namespace: systemNamespace,
+    type: "service",
+    yaml: createHttpServiceFile(),
+  });
+});
+
+test.afterAll(async () => {
+  await deleteFile({
+    namespace: systemNamespace,
+    path: systemServiceName,
+  });
+  if (systemNamespaceAlreadyExists === false) {
+    await deleteNamespace(systemNamespace);
+  }
+});
+
```

Comment:
I also thought about moving the setup and cleanup directly into the one test, but this would come with the downside that the cleanup will not run when the test fails. Even if the test is very short and clean, if it fails at some point, it would be very hard to debug if the cleanup does not run.

But what moving the test from `list.spec.ts` into a new `systemServcicesList.spec.ts` file and keeping the current setup and cleanup.

Reply:
> But what moving the test from list.spec.ts into a new systemServcicesList.spec.ts file and keeping the current setup and cleanup.

Good point... and good idea. Alternatively, you could use a describe block as described here: https://playwright.dev/docs/api/class-test#test-after-all